config/locales/en/admin.yml

```diff
@@ -42,7 +42,10 @@ en:
           means_test_review_phase_one: Enable changes for MTR Phase 1
           partner_means_assessment: Enable Partner Means Assessment
         hints:
-          mock_true_layer_data: Select Yes and TrueLayer data will be replaced by mock data from %{bank_transaction_filename}
+          mock_true_layer_data: >
```

Comment:
Updating this hint text to be more generic seemed like a reasonable tradeoff to make to me —especially since `bank_transaction_filename` cannot be updated through the Settings UI.